src/OpenLoco/src/Entities/Entity.h

```diff
@@ -48,7 +60,7 @@ namespace OpenLoco
         uint8_t linkedListOffset; // 0x8
         uint8_t var_09;
         EntityId id; // 0xA
-        uint16_t var_0C;
+        EntityBaseFlags var_0C;
```

Comment:
could look at naming this variable too

Reply:
The super annoying thing about these flags is that they are vehicle flags but in the entity base. I don't think any other entities use these but they might.

Reply:
I wanted to defer to the team on naming. Sometimes it seems like there could be multiple sets of flags in a struct so it's not clear which one should own the "_flags" name. I think it would be nice to be a bit more descriptive and call them "_commandFlags" for instance, if that is their purpose.

Reply:
They are a bit of a miscellaneous selection of flags annoyingly. Mostly its commands but its also just used by the ui to know if its sorted. I would probably name them vehicleFlags to make it clear that they have nothing to do with misc entities and add a comment that these should be moved to the vehicle base after full reimplementation. (In OpenRCT2 I think I just fully removed these flags and replaced them with alternative variables)